.travis.yml

```diff
@@ -2,7 +2,6 @@ dist: xenial
 language: python
 conditions: v1
 python:
-- 3.5
 - 3.6
 - 3.7
```

Comment:
could you try if there's 3.8 available already?

Reply:
It's available, while a test fails due to the fix for <https://pagure.io/python-daemon/issue/34> not being released. Should I add 3.8 and xfail the affected tests on that version?

Reply:
use this for now: https://docs.travis-ci.com/user/customizing-the-build/#rows-that-are-allowed-to-fail

Reply:
or the xfail indeed with failing test while it starts passing.